h/jinja_extensions/navbar_data_admin.py

```diff
@@ -43,6 +43,12 @@ def navbar_data_admin(request):
         "title": "Badge",
         "route": "admin.badge",
     },
+    {
+        "id": "documents",
+        "permission": Permission.AdminPage.HIGH_RISK,
```

Comment:
I marked this as "high risk" for now because it is a destructive action. In time I expect we might want to make this self-service for customer support, perhaps with some added guardrails.